Dockerfile.openpilot

```diff
@@ -14,6 +14,7 @@ COPY ./openpilot ${OPENPILOT_PATH}/openpilot
 COPY ./third_party ${OPENPILOT_PATH}/third_party
 COPY ./site_scons ${OPENPILOT_PATH}/site_scons
 COPY ./rednose ${OPENPILOT_PATH}/rednose
+COPY ./rednose_repo/site_scons ${OPENPILOT_PATH}/rednose_repo/site_scons
```

Comment:
what's this for? we don't do this for cereal or opendbc

Reply:
I've added custom scons method for building these filters: https://github.com/commaai/rednose/pull/37/files#diff-736583406e8e7718dade1af949f9bc4ba08f68506692ae85471fa53566664936 - so i could reuse it in xx, independently from op